tests/js/spec/components/organizations/timeRangeSelector/dateRange.spec.jsx

```diff
@@ -1,345 +0,0 @@
-import MockDate from 'mockdate';
```

Comment:
Mocking out `react-date-range` broke all these tests, so I went about rewriting them. But since this component is never used outside of `<TimeRangeSelector />`, I thought it better move the test cases to that test file.